src/test/java/org/kpmp/geneExpression/SCRNAGeneExpressionValueTest.java

```diff
@@ -0,0 +1,54 @@
+package org.kpmp.geneExpression;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class SCRNAGeneExpressionValueTest {
+
+    private SCRNAGeneExpressionValue scrnaGeneExpressionValue;
+
+    @Before
+    public void setUp() throws Exception {
+        scrnaGeneExpressionValue = new SCRNAGeneExpressionValue();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        scrnaGeneExpressionValue = null;
+    }
+
+    @Test
+    public void testGettersAndSetters() throws Exception {

```

Comment:
I much prefer these in separate methods, one for each getter/setter pair. The issue with having them all one method is that you don't get a full picture of what all might be broken because as soon as one assert fails, the test is done, then you might have to go through and run/re-run it many times to fix all of the errors.